src/components/timeline/LTimeline.vue

```diff
@@ -152,9 +157,24 @@ export default {
   }
 }
 
-@media (min-width: 1400px) {
-  .LTimeline__cardContent {
-    width: 1150px;
+@media screen and (max-width: 1400px) {
+  ::v-deep .LTimeline__cardContent {
+    width: 868px;
+    margin: 0 8px;
+  }
+}
+
+@media screen and (max-width: 1280px) {
+  ::v-deep .LTimeline__cardContent {
+    width: 768px;
+    margin: 0 8px;
+  }
+}
+
+@media screen and (max-width: 600px) {
+  ::v-deep .LTimeline__cardContent {
```

Comment:
Por que dos v-deep?

Reply:
Bem notado, vou alterar

Reply:
Um ponto de atenção também é o fato de estar usando max-width, o ideal é usar o min-width pra conseguir ir escalando dos menores tamanhos para os menores, acha que rola alterar isso também?

Reply:
Vou externalizar esse componente para o shared

Reply:
Está nos planos mandar ele pra lá mesmo, não por ter regras específicas e tal, até porque ele está bem codado. É mais pelo propósito dele ser específico. Ele não é um base component saca?
Mas se tu for externalizar já riscamos aqui do épico de migração.

De qualquer forma, mesmo no Shared o ideal é usar min-width pra media query, se for pra fazer um bagulho responsivo mobile-first e tal.

Reply:
O ponto é exatamente por isso que tem o `max-width`. Porque não foi construído mobile-first, logo no desenvolvimento da time e do detalhes no geral, não havia a preocupação com responsividade. E se for mudar agora vai alterar toda a lógica de construção do componente, ai isso pode da trabalho, tlg?

Reply:
Saquei, faz sentido. Só não entendi da migração pro Shared. Vcs vão tocar dai? Se sim, vai rolar qnd?

Reply:
...